### ISSUE TITLE
[Request] VueJS tutorial

### ISSUE THREAD

**Issue by mesqueeb** [howtographql/howtographql#785]
There's currently no VueJS tutorial anymore. I have heard [rumours](https://forum.vuejs.org/t/fullstack-tutorial-with-graphql-vuejs-apollo/16884) about there being one a long time ago.

My my humble request is for a new VueJS tutorial. : )

**Comment by mesqueeb** [comment]
@almarto Thanks! I've requested the original author for an update:
https://twitter.com/Mesqueeb/status/1045556261574520832

Let's hope he responds positively!! 💖

**Comment by almarto** [comment]
I'll do with a PR if he doesn't answer, but I'll need some time as I'm pretty busy these days

**Comment by swcarlosrj** [comment]
@almarto @mesqueeb it'd be awesome, would you be still interested on creating/updating the vue-apollo tutorial? In that case, I'd get you back letting you know if that's possible & hopefully help you with it 😄

**Comment by mesqueeb** [comment]
@swcarlosrj i would definitely want to! But I don't know anything about graphQL. I was kinda waiting for the tutorial to be able to learn it. So if I need to wait for the tutorial to learn to be able to make the tutorial, that's called catch 22!!!!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by swcarlosrj** [comment]
@mesqueeb great, I'll see if it's doable and I'll notify here!

**Comment by mesqueeb** [comment]
I can't wait! ♥

**Comment by ozum** [comment]
Any update on this?

**Comment by LongLiveCHIEF** [comment]
I'm going through this today, will see how it goes. I do these in docker, and I'll continue to update my progress in the same repo I did the backend tutorials in: https://github.com/LongLiveCHIEF/docker-howtographql.

If i don't remember to update this issue, and you see vue code in that repo, someone ping me to come back and update with status. :smiley:

**Comment by LongLiveCHIEF** [comment]
Yeah, it needs some work. I'll try to log issues and submit merge requests as I go.
